jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -17,6 +17,8 @@
 @org.springframework.context.annotation.PropertySource("classpath:application.properties")
 public class OdeProperties implements EnvironmentAware {
    private Logger logger = LoggerFactory.getLogger(this.getClass());
+   private Logger data = LoggerFactory.getLogger("data");

```

Comment:
Data logger should be singleton class. Defined once and used many places. Even if you didn't make a singleton class, you should NEVER-EVER hard-code a string like ( and such a common string as "data") all over the place. "data" should be defined once as a constant and used other places. If you define the singleton class, data is defined in that class and no other user of the logger need to know what's the name of the logger.